core/src/game/sniper_monkey/model/player/fighter/attack/Arrow.java

```diff
@@ -4,12 +4,31 @@
 import game.sniper_monkey.model.player.Player;
 import game.sniper_monkey.model.world_brick.WorldBrick;
 
+/**
+ * A class of the Arrow attackobject that the huntress shoots.
+ *
+ * @author Dadi Andrason
+ *
+ * Uses Player
```

Comment:
Does it?

Reply:
yes imo since it creates a player object with the gameobject in the addhitResponse inside the constructor and uses the player object to call the takedamage method.